tests/manual/BridgeToKubernetes.md

```diff
@@ -127,7 +127,7 @@ Please test these scenarios in this order to ensure that the Bridge to Kubernete
 
 ## <b id="bridge-vscode">Bridge to Kubernetes</b>
 **Connect a service**
-1. Open `mindaro/samples/todo-app` from the Bike Sharing sample application in Visual Studio Code. 
+1. Open `Bridge-To-Kubernetes/samples/todo-app` from the Todo sample application in Visual Studio Code. 
```

Comment:
I think this path is missing stats-api subdirectory

Reply:
Updated, thank you for catching this.